action.yml

```diff
@@ -220,6 +220,30 @@ inputs:
     required: false
     default: "false"
 
+  deno_lint:
+    description: Enable or disable deno lint checks
+    required: false
+    default: "false"
+  deno_lint_args:
+    description: Additional arguments to pass to the linter
+    required: false
+    default: ""
+  deno_lint_dir:
+    description: Directory where the deno lint command should be run
+    required: false
+  deno_lint_extensions:
+    description: Extensions of files to check with deno lint
+    required: false
+    default: "*"
```

Comment:
`deno lint` checks all supported files if the `[files]...` argument is not provided. This default value "*" is interpreted as not passing the `[files]...` argument.